### DISCUSSION TITLE
how much time it takes for the student pack to be opened

### DISCUSSION THREAD

**Discussion by chouaib1123** [community/community#145345] [GitHub Education]
### Hi everyone,

![image](https://github.com/user-attachments/assets/038941a6-862c-4e83-9526-3e712cb074c4)
im just wondeering how much time it takes for the student pack to be opened since my application already aproved , its been 4 days now

**Comment by davevad93** [comment]
Hi @chouaib1123 ,

When the benefits become available, the status label will turn purple and change to:

- `Congrats! Your request for @chouaib1123 was approved and your benefits are now available.`

The waiting time may vary depending on different factors, such as the volume of requests; it usually takes a few days, but sometimes it can take a little longer, even a couple of weeks.

For further reference you can look at these discussions: 
#132790 , #131665 , #132470 , #133531, #134279, #135849, #136887, #138721, #139268, #140287

And also the [Student Developer Pack Application FAQs](https://github.com/orgs/community/discussions/111352) :

- [How long after I submit my application will it be approved?](https://github.com/orgs/community/discussions/111352#how-long-after-i-submit-my-application-will-it-be-approved)
- [How long after I’ve been approved will I receive my academic benefits?](https://github.com/orgs/community/discussions/111352#how-long-after-ive-been-approved-will-i-receive-my-academic-benefits)

**Reply by chouaib1123** [reply]
Thx you i do understand now its only becaus my profil already aproved and only message is your pack will be avaailable soon i tooked 4 days thats why i was wondering

**Reply by davevad93** [reply]
You're welcome @chouaib1123 don't worry, you'll get your benefits soon! 👍
